pyomo/contrib/doe/fim_doe.py

```diff
@@ -1015,6 +1080,20 @@ def __create_doe_model(self):
         m.t_set = Set(initialize=time_set)
         m.tmea_set = Set(initialize=self.measurement_timeset[0])
 
+        # reorganize measurement names with
```

Comment:
Should we add a private method named `flatten_measurements`? This would allow us to test it better.